Core_Pregnancy/PregnancyGameController.cs

```diff
@@ -47,6 +48,21 @@ protected override void OnDayChange(int day)
                 // 1 day = 1 week in AI
                 // At start of each week increase pregnancy week counters of all pregnant characters
                 ApplyToAllDatas((heroine, data) => AddPregnancyWeek(data));
+                SetCharsDangerousDay();
+            }
+
+            //In AI we set Dangerous day by 20% chance on any new day
```

Comment:
Not quite how it works but good enough I guess :)
An alternative would be to keep track of which day of her cycle the heroine is on and increment it on each day. This would save to the card. You could have consecutive risky or safe days then.
Alternative way to get consecutive days would be to skew the random chance based on if the character is currently on safe or dangerous day, so it becomes more likely to stay on the same type as current.

Reply:
Also you could bring over the risky day settings from KK now that this is a thing (always/never risky stuff).

Reply:
I wasn't able to find a date time property in any game class that tracked days in game.  Only hours like 0-24 then back to 0.  Which is why I went with the random day approach.  Consecutive days should be pretty easy to add.

Ill take a look at that risky day setting as well.